Source/Core/Core/HW/AudioInterface.cpp

```diff
@@ -191,7 +194,10 @@ void RegisterMMIO(MMIO::Mapping* mmio, u32 base)
         {
           DEBUG_LOG(AUDIO_INTERFACE, "Change AIDFR to %s", tmpAICtrl.AIDFR ? "32khz" : "48khz");
           m_Control.AIDFR = tmpAICtrl.AIDFR;
-          g_AIDSampleRate = tmpAICtrl.AIDFR ? 32000 : 48000;
+          if (SConfig::GetInstance().bWii)
+            g_AIDSampleRate = tmpAICtrl.AIDFR ? 32000 : 48000;
+          else
+            g_AIDSampleRate = tmpAICtrl.AIDFR ? 32029 : 48043;
```

Comment:
Why is there the difference between 32029 on this line and 32028 on line 188?

Reply:
Because I'm stupid.  The actual value in a decimal, but modifying Dolphin to be able to use a decimal would be a total pain in the ass.  I can fix it if there's any real interest in this change.

Reply:
I'd prefer if both values matched at least, not concerned about decimals here.